homework-1/src/Point.ts

```diff
@@ -0,0 +1,34 @@
+export class Point {
+    public x: number;
+    public y: number;
+
+    constructor();
+    constructor(x?: number, y?: number) {
+        this.x = x || 0;
+        this.y = y || 0;
+    }
+
+    toString(): string {
+        return `(${this.x}, ${this.y})`;
+    }
+
+    distance(): number;
+    distance(x?: Point): number;
+    distance(x?: any, y?: number): number {
```

Comment:
Please avoid 'any' type. In this case x : Point | number

Reply:
Fixed